cookietemple/lint/domains/cli.py

```diff
@@ -13,8 +13,11 @@
 
 
 class CliPythonLint(TemplateLinter, metaclass=GetLintingFunctionsMeta):
+
     def __init__(self, path):
         super().__init__(path)
+        self.blacklisted_sync_files = [('requirements', 'requirements.txt'), ('requirements_dev', 'requirements_dev.txt'), ('changelog', 'CHANGELOG.rst')]
```

Comment:
Again, this is the first feature that does this. I don't like it, really. I still prefer consistency over this (before we rewrite it).